src/containers/offer-page/chat-dialog-window/ChatDialogWindow.tsx

```diff
@@ -6,6 +6,7 @@ import {
   useRef,
   FC
 } from 'react'
+import { useNavigate } from 'react-router-dom'
```

Comment:
same here, move it above

Reply:
> same here, move it above

thanks for your advice, moved above near custom hooks